### PR TITLE
fix: Use Yarn commands instead of NPM commands

### PR DESCRIPTION
**Pull request by Panquesito7**
#### Description of the change

- Use Yarn commands instead of NPM commands in the `.gitpod.yml` file.

#### Checklist

- [x] Added description of the change
- [x] I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/master/CONTRIBUTING.md)
- [x] Search previous suggestions before making a new PR, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/89"></a>

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
 tasks:
-  - name: npm
-    init: npm install
-    command: npm run start
+  - name: yarn
+    init: yarn install
```
Comment (krishguptadev) [inline .gitpod.yml:4]:
```suggestion
    init: yarn
```

File: .gitpod.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+    command: yarn run start
```
Comment (krishguptadev) [inline .gitpod.yml:5]:
```suggestion
    command: yarn start
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]
This is how yarn commands work 👇

**Comment by Panquesito7** [comment]
> This is how yarn commands work 👇

I used those commands and it worked. 🤔

**Comment by vinzvinci** [comment]
> > This is how yarn commands work 👇
> 
> I used those commands and it worked. 🤔

I think most of us prefer the way @krishguptadev suggested, we are using that commands in dev protocol

**Comment by krishguptadev** [comment]
> > This is how yarn commands work 👇
> 
> I used those commands and it worked. 🤔

Actually, those commands are legacy alternative ways. (names added just to make the lives of people who used to use npm earlier easier)

**Review by krishguptadev — APPROVED** [review]

**Review by shuklaritvik06 — APPROVED** [review]

**Review by vinzvinci — COMMENTED** [review]
Great collaboration everyone!

**Pull request merged**